### PR TITLE
Fix ids and buttons in tables

### PR DESCRIPTION
**Pull request by spectranaut**
Ok so through experimentation I started to understand what was happening, but I have some questions!

Maybe @michael-n-cooper can answer these questions:
1. I'd like to understand the purpose of having a "published" ReSpec'd document, and, frankly what to call it. What I mean is when you run the script locally: `respect index.html out.html`, the `out.html` document. For now I'll call it the "published" document (I also have called it "compiled"). So I'm curious: what is the point of having this published version of the document, rather than just opening the original index.html document in a browser? I guess the point is to view it offline? Are there other reasons? For example, to have less scripts run when you open the published document? 
2. In the published version, can we have javascript and buttons and interactive elements? Just making sure it's not suppose to be eventually printed and then put in a file cabinet or something.

What I noticed when making this fix:
1. After the document is published, any event handlers added in the an "PreProcess" or "PostProcess" script, DISAPPEAR. I guess because to get the published document, you open the original document in a browser and let all the code run, then you just save that new html document? So the event handlers don't persist? I  guess the "PreProcess" and "PostProcess" are just for rearranging the DOM?
2. When the document is respec'd, any included scripts are run, then when the "published" document is loaded, all the included scripts are run again (except for the PreProcess and PostProcess stuff). Thus, leading to the weird duplication.

What I did in this PR:
1. I moved all the event handlers into the `document.addEventListener("DOMContentLoaded"...`, so they are run when you open either the published document OR the not published "index.html" document.
2. Removed the "hide column" logic but we can add it back if anyone wants it.

tldr: Any DOM changes need to happen via PreProcess or PostProcess, event handlers need to be added on DOMContentLoaded. I should make some comments to make this clearer.

### CODE REVIEW

**Comment by spectranaut** [comment]
Closing: https://github.com/w3c/aria-common/pull/84
